### PR TITLE
Añadido soporte a Salamandra TA 2B

### PR DESCRIPTION
**Pull request by carlosep93**
Añadido soporte para salamandra TA.  Actualmente el modelo sigue en privado, para probarlo hay que hacer login en huggingface usando huggingface-cli antes de ejecutar la api.

Para ejecutar la api:
```
python main.py --load MULTI-MULTI-salamandra     
```
Para hacer una request: 
```
curl --location --request POST 'http://127.0.0.1:8000/api/v1/translate' \\n--header 'Content-Type: application/json' \\n--data-raw '{"src":"en", "tgt":"it", "text":"c'\''This is a test"}'
```
src y tgt pueden ser cualquier par de la lista languages en config.json.

### CODE REVIEW

**Comment by ankush13r** [comment]
Hola @carlosep93, ¡gracias por el pull request!

He estado revisando la api en Docker y va todo bien, aunque, he encontrado el siguiente warning:

**WARNING**
```
The attention mask and the pad token id were not set.
As a consequence, you may observe unexpected behavior. Please pass your input's attention_mask to obtain reliable results.
Setting pad_token_id to eos_token_id: 2 for open-end generation.
```
¿Crees que deberíamos hacer algún ajuste para resolverlo, o podemos ignorarlo?

Cuando confirmes que el warning no es importante, procedo con el pull request.

**Comment by carlosep93** [comment]
Hola [ankush13r](https://github.com/ankush13r)!

Ese warning solo aparece porque si traducimos una sola frase no se añade padding. Es normal y no afecta a los resultados.

**Review by ankush13r — APPROVED** [review]
He revisado el código y todo parece estar bien. Solo he notado que, en caso de que alguien cargue `Salamandra` y otro modelo`es-ca`, por defecto se usa `es-ca,`, aunque el usuario haya especificado multi_model en su request.
Son cambios mínimos, los realizaré en otra rama y te avisaré cuando esté listo.
